admin/sql/updates/2021-05-04-add-report-users-table.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
+CREATE TABLE reported_users (
+    id                  SERIAL,
+    reporter_user_id    INTEGER NOT NULL, -- FK to "user".id of the user who reported
+    reported_user_id    INTEGER NOT NULL, -- FK to "user".id of the user who was reported
+    reported_at         TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+    reason              TEXT
```

Comment:
This migration has already been run on the database so that we could test this on beta. But at that time we did not have a `reason` column, there are two ways to about this.
1. Add another migration script to add the `reason` column using `ALTER TABLE`.
2. Add a `DROP TABLE IF EXISTS` at the beginning of this script.